26-register_registration/creator-navigates-to-contributor-tab.feature

```diff
@@ -19,52 +19,67 @@ Feature: Creator navigates to Contributors tab
-      | Journal           | JournalBooklet                | Add Author, Add Contributor                 |
-      | Journal           | JournalConferenceAbstract     | Add Author, Add Contributor                 |
-      | Report            | ReportBasic                   | Add Author, Add Contributor                 |
-      | Report            | ReportPolicy                  | Add Author, Add Contributor                 |
-      | Report            | ReportResearch                | Add Author, Add Contributor                 |
-      | Report            | ReportAbstractCollection      | Add Author, Add Contributor                 |
-      | Report            | ReportWorkingPaper            | Add Author, Add Contributor                 |
-      | Presentation      | ConferenceLecture             | Add Author, Add Contributor                 |
-      | Presentation      | ConferencePoster              | Add Author, Add Contributor                 |
-      | Presentation      | Lecture                       | Add Author, Add Contributor                 |
-      | Presentation      | OtherPresentation             | Add Author, Add Contributor                 |
-      | Artistic          | ArtisticDesign                | Add Contributor                             |
-      | Artistic          | Architecture                  | Add Contributor                             |
-      | Artistic          | PerformingArts                | Add Contributor                             |
-      | Artistic          | MovingPicture                 | Add Contributor                             |
-      | Artistic          | MusicPerformance              | Add Contributor                             |
-      | Artistic          | LiteraryArts                  | Add Contributor                             |
-      | Artistic          | VisualArts                    | Add Contributor                             |
-      | MediaContribution | MediaFeatureArticle           | Add Author, Add Contributor                 |
-      | MediaContribution | MediaReaderOpinion            | Add Author, Add Contributor                 |
-      | MediaContribution | MediaInterview                | Add Contributor                             |
-      | MediaContribution | MediaBlogPost                 | Add Author, Add Contributor                 |
-      | MediaContribution | MediaPodcast                  | Add Contributor                             |
-      | MediaContribution | MediaParticipationInRadioOrTv | Add Contributor                             |
-      | ResearchData      | DataManagementPlan            | Add Author, Add Contributor                 |
-      | ResearchData      | Dataset                       | Add Contributor                             |
-      | OtherPublication  | Map                           | Add Contributor                             |
+      | RegistrationType              | AddContributorButtons                       |
```

Comment:
Det er vel ikke mer en en knapp for å legge til contributors?

Reply:
fjerner dette scenarioet